src/Avalonia.Controls/Presenters/ItemVirtualizerSimple.cs

```diff
@@ -97,7 +97,11 @@ public override double ViewportValue
         /// <inheritdoc/>
         public override Size MeasureOverride(Size availableSize)
         {
-            var window = Owner.GetVisualRoot() as TopLevel;
+            var maxAvailableSize = (Owner.GetVisualRoot() as WindowBase)?.PlatformImpl?.MaxClientSize;
+            if (!maxAvailableSize.HasValue)
+            {
+                maxAvailableSize = (Owner.GetVisualRoot() as TopLevel)?.ClientSize;
```

Comment:
I'm not sure this logic is correct: won't this prevent a window that has a `SizeToContent` of anything other than `Manual` from resizing to fit the visible items?